src/ServiceControl/Operations/Msmq/MsmqAuditQueueImporter.cs

```diff
@@ -27,12 +27,6 @@ public MsmqAuditQueueImporter(IDocumentStore store, IBuilder builder, IDequeueMe
             this.store = store;
             this.builder = builder;
             enabled = receiver is MsmqDequeueStrategy;
```

Comment:
Makes sense to me but I wanted this particular change to be as small as possible without making any other assumptions outside of the actual area being fixed